spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SparkWriteOptions.java

```diff
@@ -68,4 +68,11 @@ private SparkWriteOptions() {
   // Controls whether to take into account the table distribution and sort order during a write operation
   public static final String USE_TABLE_DISTRIBUTION_AND_ORDERING = "use-table-distribution-and-ordering";
   public static final boolean USE_TABLE_DISTRIBUTION_AND_ORDERING_DEFAULT = true;
+
+  // Identifies snapshot from which to start validating conflicting changes
+  public static final String VALIDATE_FROM_SNAPSHOT_ID = "validate-from-snapshot-id";
+
+  // Isolation Level for DataFrame calls.  Currently supported by overwritePartitions
+  public static final String ISOLATION_LEVEL = "isolation-level";
+  public static final String DYNAMIC_OVERWRITE_LEVEL_DEFAULT = "snapshot";
```

Comment:
Like I said above, I think the default should be no validation. I'd remove this constant.

Reply:
Done